framework/tests/annotationclassloader/Makefile

```diff
@@ -47,34 +45,35 @@ demo1:
 demo2:
 	@echo "***** This command is expected to produce an error on line 7:"
 	$(JAVAC) \
-	  -J-Xbootclasspath/p:$(FRAMEWORKJAR):$(STUBPARSERJAR):$(JAVACJAR) \
-	  -classpath :$(PROJECTDIR) \
+	  -processorpath $(FRAMEWORKJAR) \
```

Comment:
Are these "fat" jars or do they just contain the individual project files? It seems a bit odd that the different demos in this file use different `processorpath`s, even though they probably all have the same dependencies. It would probably be best if all of them depend on the `build` versions and you add a new variable for the union of all of them.

Reply:
They are fat jars.  The tests are suppose to use different processorpaths to test loading annotations from a jar file and loading annotations from classes.